README.rst

```diff
@@ -0,0 +1,98 @@
+===================================================
+tuesday - The Ultimate EoR Simulation Data AnalYser
+===================================================
+A collection of lightcone postprocessing tools such as calculating power spectra, plotting, and making line intensity maps.
+
+|PyPI| |Status| |License| |Version| |Python Version| |Docs| |Code Style| |Codecov|
+
+.. |PyPI| image:: https://badgen.net/pypi/v/tuesday-eor/
+   :target: https://pypi.org/project/tuesday-eor
+   :alt: PyPI version
+.. |Status| image:: https://badgen.net/github/status/tuesday.svg
+    :target: https://pypi.org/project/tuesday-eor
+    :alt: Status
+.. |License| image:: https://img.shields.io/badge/License-MIT-yellow.svg
+    :target: https://opensource.org/licenses/MIT
+    :alt: License
+.. |Version| image:: https://badgen.net/pypi/v/tuesday-eor/
+    :target: https://pypi.org/project/tuesday-eor
+    :alt: Version
+.. |Python Version| image:: https://img.shields.io/pypi/pyversions/tuesday-eor.svg
+    :target: https://pypi.python.org/pypi/tuesday-eor/
+    :alt: Python Version
+.. |Docs| image:: https://readthedocs.org/projects/tuesday/badge/?version=latest
+    :target: http://tuesday.readthedocs.io/?badge=latest
+    :alt: Documentation Status
+.. |Code Style| image:: https://img.shields.io/badge/code%20style-black-000000.svg
+    :target: https://github.com/psf/black
+.. |Codecov| image:: https://codecov.io/gh/21cmfast/tuesday/branch/main/graph/badge.svg
+    :target: https://app.codecov.io/gh/21cmfast/tuesday
+    :alt: Code Coverage
+
+Installation
+============
+
+``tuesday`` is available on PyPI and can be installed with standard
+tools like ``pip`` or ``uv``::
+
+    $ pip install tuesday-eor
+
+or::
+
+    $ uv pip install tuesday-eor
+
+If you are developing ``tuesday``, we recommend using a virtual environment.
+You can create a new environment with ``uv``::
+
+    $ uv sync
+    $ source .venv/bin/activate
+
+Contribute
+==========
+
+``tuesday`` is meant to be the collection of all these useful functionalities of which everyone has their own implementation, such as the power spectrum calculation.
+To contribute to ``tuesday``, first find where your code belongs:
+if your code can be written in a simulator-independent manner (preferred), it goes into ``core``.
+On the other hand, if it requires a simulator-dependent input, then it goes into ``simulators/your_simulator``.
+
```

Comment:
You could add something like "In some cases, multiple implementations of similar functionality might exist -- one general-purpose tool that can act on raw arrays, and other higher-level implementations that act on bespoke simulator outputs, that call the more general methods under the hood."